routes/login.js

```diff
@@ -0,0 +1,99 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const config = require("../config");
+const jwtService = require("../services/JwtService");
+
+const userService = require("../services/UserService");
+
+router.get("/", (req, res) => {
+    userService
+        .getAllUsers()
```

Comment:
Why does `login.js` have `getAllUsers()` method? I think this route should be dedicated only towards the auth and login process. Maybe `auth.js` will be a better name for this.

Reply:
It was just for testing purpose and would be removed